planning/behavior_path_planner/include/behavior_path_planner/scene_module/goal_planner/goal_planner_module.hpp

```diff
@@ -230,6 +232,18 @@ class GoalPlannerModule : public SceneModuleInterface
     const Pose & search_start_offset_pose, PathWithLaneId & path) const;
   PathWithLaneId generateStopPath();
   PathWithLaneId generateFeasibleStopPath();
+
+  /**
+   * @brief Generate a stop point in the current path based on the vehicle's pose and constraints.
+   *
+   * This function generates a stop point in the current path. If no lanes are available or if
+   * stopping is not feasible due to constraints (maximum deceleration, maximum jerk), no stop point
+   * is inserted into the path.
+   *
+   * @return PathWithLaneId The modified path with the stop point inserted. If no stop point is
+   * inserted, returns the original path.
+   */
+  PathWithLaneId generateStopPointInCurrentPath();
```

Comment:
this function name is a little unclear because the return value is path.

```suggestion
  PathWithLaneId generateStopCurrentPath();
```
:thinking:

Reply:
how about generateStopInsertedCurrentPath?generateStopPath?

Reply:
changed in https://github.com/autowarefoundation/autoware.universe/pull/4927/commits/dbc6b460638d80ae3c6c596ce2e45b6fca964be1